src/prompts.ts

```diff
@@ -31,7 +31,8 @@ export default async function runSummaryPrompt(
 - Keep in mind that the 'Original title', 'Original description' and 'Commit messages' sections may be partial, simplistic, non-informative or out of date. Hence, compare them to the PR diff code, and use them only as a reference.
 - The generated title and description should prioritize the most significant changes.
 - When quoting variables or names from the code, use backticks (\`).
-- Return a summary for each single affected file or "no summary" if there is nothing to summarize.
+- Return a summary for each single affected file or if there is nothing to summarize simply use the status of the change (ie. "New file").
+- Start the overview with a verb at past tense like "Started", "Commented", "Generated" etc...
```

Comment:
The instructions for generating summaries could be clearer. Consider specifying what kind of summaries are expected for different types of changes (e.g., new files, modified files).